FrontPage/Client.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import Apollo
+import AppAuth
+
+let config = Config()
+let syncUrl = config.getConfiguration("sync-app")
+let syncConfig = syncUrl.config ?? ["error": JSONValue.bool(false)]
+let syncWs = syncConfig["websocketUrl"]?.getString() ?? "error";
+
+class Client{
+  static let instance = Client()
+  let auth = AuthViewController()
+  
+  private(set) lazy var client: ApolloClient = {
```

Comment:
given that this class is called `Client`, I would suggest that we don't use the same for the variable here. Perhaps just call it `apollo` or `apolloClient` instead.